sdks/go/pkg/beam/core/runtime/exec/combine_test.go

```diff
@@ -60,9 +170,30 @@ func TestCombine(t *testing.T) {
 	if err := p.Down(context.Background()); err != nil {
 		t.Fatalf("down failed: %v", err)
 	}
+}
 
-	expected := makeKV(42, 21)
-	if !equalList(out.Elements, expected) {
-		t.Errorf("pardo(sumFn) = %v, want %v", extractKeyedValues(out.Elements...), extractKeyedValues(expected...))
+func mergeFn(a, b int) int {
+	return a + b
+}
+
+type MyCombine struct{}
+
+func (*MyCombine) AddInput(k int64, a int) int64 {
+	return k + int64(a)
+}
+
+func (*MyCombine) MergeAccumulators(a, b int64) int64 {
+	return a + b
+}
+
+func (*MyCombine) ExtractOutput(a int64) int {
+	return int(a)
+}
+
```

Comment:
It'd be nice to have a more elaborate combiner with differentiated input, accumulator, and output types.

Reply:
Done.